edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/util/aggrigation/IAggrigator.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Util.Aggrigation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    public interface IAggrigator
+    {
+        public void PutValue(double value);
+
+        public double GetAggrigate();
+
+        public IAggrigator New();
```

Comment:
Nit: "new" is the common factory function name in rust, but not C# right? Is it "Create"? I don't remember...

Reply:
I couldn't find a consensus online